### ISSUE TITLE
panicked at 'bug in tokio-rusqlite, please report: "oneshot send"' / bug in tokio-rusqlite, please report: Ok(())

### ISSUE THREAD

**Issue by davehorner** [programatik29/tokio-rusqlite#8]
Hello,
I am getting panic notices while running hyper/tokio.  It doesn't report the same error consistently.  I get messages that look like:
```
panicked at 'bug in tokio-rusqlite, please report: "oneshot send"'
thread '<unnamed>' panicked at '\n' , github.com-1ecc6299db9ec823\tokio-rusqlite-0.1.0\src\lib.rs:254:36
thread '<unnamed>' panicked at 'bug in tokio-rusqlite, please report: Ok(())', \github.com-1ecc6299db9ec823\tokio-rusqlite-0.1.0\src\lib.rs:254:36

```
Aside from the error messages going to the console, the code continues to work.
Thanks for the code!

**Comment by programatik29** [comment]
Hello,
This was fixed in #2. Updating the version should do.

Have a nice day.

**Comment by davehorner** [comment]
Updated.  No more panics.  Thank you!
